modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/exceptions.jspf

```diff
@@ -98,10 +94,12 @@
 			if (Validator.isNull(fieldLabelValue)) {
 				fieldLabelValue = msvv.getFieldName();
 			}
+
+			showExpirationDateError = Objects.equals(msvv.getFieldName(), "expirationDate");
```

Comment:
@rebsilva, how about we optimize it with the following code?
```
boolean showExpirationDateError = Objects.equals(msvv.getFieldName(), "expirationDate");
```
Using the code above, I believe that we can remove the new lines used just to declare the variable `showExpirationDateError`. What do you think about it, Beca?

Reply:
Hello @natocesarrego! Thanks for the review!
I think that in this case we need to declare the variable before, because we use it on `<c:if test="<%= !showExpirationDateError && ddmFormAdminDisplayContext.isFormExpired(ddmFormAdminDisplayContext.getDDMFormInstance()) %>">` at line 159 to display the warning. If we declare the `showExpirationDateError` inside the `DDMFormValuesValidationException.MustSetValidValue.class` I believe it will change the expected behaviour of the warning.